### ISSUE TITLE
Advanced ML code in python

### ISSUE THREAD

**Issue by saratheruvaram** [saratheruvaram/siva#4]
Sure! Here's an example of a Python program that demonstrates advanced machine learning techniques using the scikit-learn library:

```python
import numpy as np
from sklearn.datasets import load_iris
from sklearn.model_selection import train_test_split
from sklearn.preprocessing import StandardScaler
from sklearn.svm import SVC
from sklearn.metrics import accuracy_score

# Load the Iris dataset
iris = load_iris()
X = iris.data
y = iris.target

# Split the dataset into training and testing sets
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42)

# Scale the features
scaler = StandardScaler()
X_train = scaler.fit_transform(X_train)
X_test = scaler.transform(X_test)

# Train a Support Vector Machine (SVM) classifier
svm = SVC()
svm.fit(X_train, y_train)

# Make predictions on the testing set
y_pred = svm.predict(X_test)

# Calculate the accuracy of the classifier
accuracy = accuracy_score(y_test, y_pred)
print("Accuracy:", accuracy)
```

This program uses the Iris dataset, splits it into training and testing sets, scales the features using standardization, trains a Support Vector Machine (SVM) classifier, makes predictions on the testing set, and calculates the accuracy of the classifier.
